README.md

```diff
@@ -42,6 +42,10 @@ Arguments and options:
                                          will write into this file instead of
                                          printing to the console. (default:
                                          /tmp/mygraph.dot)
+ --outputXMIFile=FILE                  : Path to the XMI output file. If given,
+                                         will write into this file instead of
+                                         printing to the console. (default:
+                                         /tmp/mygraph.dot)
```

Comment:
If there is a default value, I suppose it is not a .dot file?

Reply:
Also, I would not say that it writes to a file *instead of* printing to the console, since the XMI was never printed to the console in the first place?

Reply:
And also, I would be more precise than `XMI output file` and I would explain the user that the serialized file contains the intermediate EMF model representing the features/plugins and their dependencies.

Reply:
Ah indeed, I forgot to modify those. I've just made the requested changes.

> Also, I would not say that it writes to a file _instead of_ printing to the console, since the XMI was never printed to the console in the first place?

So I've changed it to `will write into this file instead of printing the DOT graph to the console`. I also modified a little the description of `--alwaysPrint` to `If set, the output DOT graph is printed even if an output file is given` since the xmi is never printed out even if we use the `--alwaysPrint` with `--outputXMIFile` option. 

> Will there be another PR with the extension made to the metamodel?

Yes, I'll send another PR soon :)